modules/machine-feature-agnostic-capacity-reservation.adoc

```diff
@@ -45,15 +45,24 @@ spec:
   template:
     spec:
       capacityReservationId: <capacity_reservation> # <1>
-      marketType: <market_type> # <2>
+      capacityReservationPreference: <reservation_preference> # <2>
+      marketType: <market_type> # <3>
 # ...
 ----
 <1> Specify the ID of the 
 ifdef::azure[Capacity Reservation group]
 ifdef::aws[Capacity Block for ML or On-Demand Capacity Reservation]
 that you want to deploy machines on.
 ifdef::aws[]
-<2> Specify the market type to use.
+<2> Specify your preferred capacity reservation behavior.
+The following values are valid:
+`capacity-reservations-only`:: Use this option to require a matching capacity reservation.
+If no matching capacity reservation is available, the instance fails to launch.
+`open`:: Use this option to allow using an open capacity reservation that matches the availability zone and instance type.
+If no matching capacity reservation is available, the instance falls back to On-Demand.
```

Comment:
I think it is better to delete this line because although `capacityReservationPreference` is aimed to configure the instance to run as an On-Demand instance, we can also set `capacityReservationPreference: Open` on a Spot instance, in which case the instance created is Spot instead of On-Demand.